dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/SdkFeatureTest.kt

```diff
@@ -266,8 +268,10 @@ internal class SdkFeatureTest {
 
     // region FeatureScope
 
-    @Test
+    @ParameterizedTest
```

Comment:
not sure if we need a parametrized test here, given that we don't assert anything related to the `forceNewBatch` value.

We can simply use `BoolForgery` on the `forceNewBatch` test parameter. And coverage will be the same, because the only place where code path is different depending on the `forceNewBatch` parameter is `BatchFileOrchestrator`.

Reply:
Yeah but I rather make sure we test with both true/false value.

Reply:
IMO this would make sense if some assertion depends on that, which is not the case in most of the test. Anyway we already have kinda check of the proper flow by doing `whenever(object.methodCall(parameter))`, because if the parameter is not the one we provided in `whenever`, mock won't work and test will break anyway.

But I'm not pushing for it :)